scripts/test_docker_compose.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
-set -ex
+try_cmd() {
+    for i in $(seq 10); do
+        ./iroha_client_cli "$@"
+        [ $? = 0 ] && return 0
+        sleep 0.3
+    done
+    exit 1
+}
```

Comment:
Bash scripting is extremely error prone if we start having such scripts then it is better to rewrite them in higher level programming languages. Maybe we should rewrite this test process in Rust https://doc.rust-lang.org/std/process/struct.Command.html#method.output

Or if the compilation takes too long we might consider Python I guess, but I don't think having several languages will be good for the codebase. So as much as possible let's stick to Rust. 

My suggestion is to not modify `test_docker_compose.sh` then in this PR. And if we need to improve it, then rewrite it in Rust in a separate PR.